website/pages/docs/reference/cli/cloudquery_validate-config.md

```diff
@@ -33,6 +33,7 @@ cloudquery validate-config ./directory ./aws.yml ./pg.yml
 
 ```
       --cq-dir string            directory to store cloudquery files, such as downloaded plugins (default ".cq")
+      --invocation-uuid uuid     useful for when using Open Telemetry integration for tracing and logging to be able to correlate logs and traces through many services (default 20ca6cb3-d72e-479c-b789-29ee71f082c8)
```

Comment:
Surely these default values are misleading and will change with each gen, too.

Reply:
Didn't even notice that. Will look into how to fix that

Reply:
fixed in [b13a660](https://github.com/cloudquery/cloudquery/pull/18878/commits/b13a6600b466ba72b763586e2e03f57cf57050a4)